### ISSUE TITLE
Improve experience with prompt templates (feature)

### ISSUE THREAD

**Issue by stepelu** [lmg-anon/mikupad#43]
First of all thank you for sharing this work, it's great for conducting experiments :)
As of now, prompt templates are quite heavy on the eye when multiple messages are exchanged between the user and the LLM.
It would be great if there was an option to automatically collapse all text matching a set of strings to say `<…>`, so that it doesn't visually clutter the screen.
The text would expand and be editable when clicking.
The idea is a bit like https://github.com/KeitaNakamura/tex-conceal.vim...

**Comment by jukofyork** [comment]
Yeah, or even the option to make the templates grey / low contrast so they don't stick out so much would be good.
